specs/v1.yaml

```diff
@@ -1343,9 +1343,6 @@ components:
           type: string
           enum:
             - Initiated
-            - Processing
```

Comment:
technically a breaking change

Reply:
yes but it was removed in our grant statuses page already in a past version